Started by upstream project "[8mha:////4GD9uaz4zbgQKoz8w38ohKuvUeYmdLoFkyctuh1Y5p1WAAAAoR+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAz+EgZR/az8JH3nnNTEvPjy/KLs4oLE5FR9APX8eN/KAAAA[0mClean_workspace" build number [8mha:////4M02vmIvo5CnEAk0lOzVu0aGELfk2Ms46bww7izamopxAAAAoR+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLQIwSBjH9rPwkfeec1MS8+PL8ouzigsTkVH1jAMQ5yLzLAAAA[0m3
originally caused by:
 Started by upstream project "[8mha:////4Bera1cDne2h6MBNh29kJJy0hfuZKsy6qBXen/DTuE1nAAAAqR+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzxEgZZ/az8JH2XzMTigvyixPjikvyi1PiS/Pj0osSCDH0AOq6BxNIAAAA=[0mDiaspora_store_to_graph" build number [8mha:////4MkRCA31IvXKJXj0xqitXYlsPLJlZBQULZfJFoBZuvyRAAAAqx+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAymEgZ5/az8JH2XzMTigvyixPjikvyi1PiS/Pj0osSCDH1jUwDPKv3H1AAAAA==[0m35
 originally caused by:
  Started by upstream project "[8mha:////4DqpYtamEh1Kj6fZelSZDYC8OqIresBqvMdk/sI8VmdXAAAAoh+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAyBEgYx/az8JP2UzMTigvyixPiilLT43MQCfQBfAkAKywAAAA==[0mdiaspora_rdf_map" build number [8mha:////4HnqGjKnwkroCxWlQOjaJiagxLzQzWxl3JyA5f2FrXPUAAAApB+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAymEgYJ/az8JP2UzMTigvyixPiilLT43MQCfWNDAPmJpWjNAAAA[0m31
  originally caused by:
   Started by user [8mha:////4IGVMtvk6iK1hwW+PI4nElqD5WuZBmgWM1foNK6n5D+9AAAAlx+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzuEgZu/dLi1CL9ovyMzBIAOOgHy8AAAAA=[0mrohit gupta
Running as SYSTEM
Building in workspace /var/jenkins_home/workspace/query_diaspora
[query_diaspora] $ /bin/sh -xe /tmp/jenkins218946705460117062.sh
+ python3 query.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/SPARQLWrapper/Wrapper.py", line 926, in _query
    response = urlopener(request)
  File "/usr/lib/python3.9/urllib/request.py", line 214, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.9/urllib/request.py", line 523, in open
    response = meth(req, response)
  File "/usr/lib/python3.9/urllib/request.py", line 632, in http_response
    response = self.parent.error(
  File "/usr/lib/python3.9/urllib/request.py", line 561, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.9/urllib/request.py", line 494, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.9/urllib/request.py", line 641, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: 

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/jenkins_home/workspace/query_diaspora/query.py", line 52, in <module>
    results = sparql.queryAndConvert()
  File "/usr/local/lib/python3.9/dist-packages/SPARQLWrapper/Wrapper.py", line 967, in queryAndConvert
    res = self.query()
  File "/usr/local/lib/python3.9/dist-packages/SPARQLWrapper/Wrapper.py", line 960, in query
    return QueryResult(self._query())
  File "/usr/local/lib/python3.9/dist-packages/SPARQLWrapper/Wrapper.py", line 932, in _query
    raise EndPointNotFound(e.read())
SPARQLWrapper.SPARQLExceptions.EndPointNotFound: EndPointNotFound: It was not possible to connect to the given endpoint: check it is correct. 

Response:
b'Unknown repository: Diaspora'
Build step 'Execute shell' marked build as failure
Finished: FAILURE
